src/store/index.js

```diff
@@ -22,4 +24,21 @@ export function getMessageSubject(message: Message): ?string {
   return subjectHeader && subjectHeader.value;
 }
 
+export function getMessageSender(message: Message): ?string {
+  const { headers } = message.payload;
+  const emailSender = headers.find(header => header.name === 'From');
+  return emailSender && emailSender.value;
+}
+
+export function getMessageTimestamp(message: Message): ?string {
```

Comment:
Nice job following the pattern set initially. You might consider eventually abstracting some of this function to prevent some of the repetition between `getMessageSender` and `getMessageTimestamp`